lib/global.js

```diff
@@ -1,14 +1,18 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
 
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
-  .map(function (arr) {
-    return arr[0]
-  })
-  .filter(function (url) {
-    return url.indexOf('wss://') === 0 || url.indexOf('ws://') === 0
-  })
```

Comment:
Please re-add this. See https://github.com/feross/webtorrent/issues/672#issuecomment-196614860 for reasoning.

Eventually, we'll remove this from `webtorrent-hybrid` and [`webtorrent-app`](https://github.com/feross/webtorrent-app), but we need to keep it for now.